src/app.ts

```diff
@@ -2,6 +2,8 @@ import inquirer from 'inquirer';
 import chalk from 'chalk';
 import yargs, { Argv } from 'yargs';
 import updateNotifier from 'update-notifier';
+import { STS } from 'aws-sdk';
```

Comment:
Change this to `import STS from 'aws-sdk/clients/sts'` or something to that effect. We should do this everywhere so we're not importing the entire AWS SDK.

Reply:
I can import from this and the types are correct, but the call just hangs - if I import from just aws-sdk it works 🤷

Reply:
That's interesting. We can try to fix this later.